lib/breathe/client.rb

```diff
@@ -2,7 +2,8 @@ module Breathe
   class Client
     attr_reader :api_key, :last_response
 
-    BASE_URL = "https://api.breathehr.com/v1/"
+    redacted_url = ENV.fetch("BREATHE_REDACTED_API_URL", nil)
```

Comment:
Theoretically anyone can use this gem (though in practice I think it's just us right now), and referring to a redacted API feels very dxw-specific. I suggest changing `BREATHE_REDACTED_API_URL` to just `BREATHE_API_URL`, and `redacted_url` to just `url`.